### PR TITLE
Add dump option

### PR DESCRIPTION
**Pull request by kerneldemon**
Hey,

I'd like a way to dump plain sql using the `phinx migrate` and `phinx rollback` commands, so I added a `--dump` option.

This is useful for those who would like to read raw queries before running them in production, there's a discussion about this [here](https://github.com/cakephp/phinx/issues/5).

### CODE REVIEW

File: src/Phinx/Console/Command/Migrate.php [hunk 1]
```diff
@@ -47,6 +47,7 @@ protected function configure()
              ->setDescription('Migrate the database')
              ->addOption('--target', '-t', InputOption::VALUE_REQUIRED, 'The version number to migrate to')
              ->addOption('--date', '-d', InputOption::VALUE_REQUIRED, 'The date to migrate to')
+             ->addOption('--dump', '-x', InputOption::VALUE_NONE, 'Dump query to standard output instead of executing it')
```
Comment (lorenzo) [inline src/Phinx/Console/Command/Migrate.php]: Do you mind renaming this to `dry-run` or `attempt`?

**Comment by lorenzo** [comment]
I'm not sure this solution is actually going to work for all cases, but it is a good start.

**Comment by kerneldemon** [comment]
@lorenzo the name `dry-run` sounds about right

**Comment by lorenzo** [comment]
Thanks @kerneldemon! Would you mind adding a brief explanation of this new flag to the docs?

**Comment by kerneldemon** [comment]
@lorenzo I contemplated adding a separate section that would describe the option, but since the option can only be used by two commands, I thought it would be better to just have the same text for migrate and rollback command section.

**Comment by lorenzo** [comment]
@kerneldemon duplicating the test sounds good

**Comment by kerneldemon** [comment]
@lorenzo Moved the text slightly. Is anything else missing?

**Comment by lorenzo** [comment]
Thanks for the contribution!

**Pull request merged**